src/requirements.txt

```diff
@@ -172,10 +172,13 @@ opencensus-ext-azure==1.0.7
     # via -r requirements.in
 opencensus-ext-django==0.7.4
     # via -r requirements.in
+opencensus-ext-logging==0.1.0
+    # via -r requirements.in
```

Comment:
I see no changes to `requirements.in` forgot to check in file?

Reply:
No, it's the other way around ;-). I committed `requirements.in` in commit 9e8d958c91e933d31fd8206e84cb2d1f76e35f5f (March 18th) but forgot to regenerate `requirements.txt`. In this PR I'm fixing that omission.

Reply:
;(